scout/server/blueprints/variants/views.py

```diff
@@ -69,11 +69,13 @@ def variants(institute_id, case_name):
         form.variant_type.data = variant_type
         # set chromosome to all chromosomes
         form.chrom.data = request.args.get("chrom", "")
-        form.gene_panels.data = [
-            panel["panel_name"]
-            for panel in case_obj.get("panels", [])
-            if panel["is_default"] is True
-        ]
+
+        if form.gene_panels.data == []:
```

Comment:
It somehow doesn't strike me as standard python, but it does away with the length and is not operators I suppose. I will not fail working code on style, especially if the linters and what not give it a pass!